include/hmap/dynamic-hmap.hpp

```diff
@@ -300,9 +300,10 @@ class DynamicHMap {
 	DynamicHMap(Args&& ...args)
 	: map_(std::forward<Args>(args) ...) {}
 	
-	
 	// Construct a DynamicHMap by using Key<V>, std::any pairs,
 	// casting Key<V> to KeyBase to perform type erasure.
+	DynamicHMap(std::in_place_t) { }
```

Comment:
This should have exactly the same behavior as the templated variadic `std::in_place_t` constructor, and is thus unnecessary.

Note that although `int foo[0]` is disallowed, `std::array<int, 0>` is a specifically allowed specialization that behaves just like any other empty collection.